pkg/controller/validate/validate.go

```diff
@@ -196,3 +219,57 @@ func projectCustomRoles(customRoles []mdbv1.CustomRole) error {
 
 	return err
 }
+
+func encryptionAtRest(project *mdbv1.AtlasProject) error {
```

Comment:
Would be better to pass only `EncryptionAtRest` struct? I think it would also make the evaluation be less verbose

Reply:
checking

Reply:
agreed, fixing in next update.

Reply:
done